packages/Confirmation/stories/examples/Showcase.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { select, text } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import L10n from "@paprika/l10n";
+import Heading from "@paprika/heading";
+import Confirmation from "../../src";
+
+const handleConfirm = () => {
+  action("Clicked the confirmation button")();
+};
+
+const handleCancel = () => {
+  action("Clicked the cancel button")();
+};
+
+const ExampleStory = () => (
+  <React.Fragment>
```

Comment:
Can you wrap this in a `<Story>` component, like most other stories are, to give it some padding?